packages/tasit-action/src/TasitAction.js

```diff
@@ -5,10 +5,11 @@ import Land from "./erc721/Land";
 import Decentraland from "./marketplace/Decentraland";
 import Estate from "./erc721/Estate";
 import Mana from "./erc20/Mana";
+import DetailedERC20 from "./erc20/DetailedERC20";
 
 export { Contract, NFT, Land, Decentraland, Estate, Mana };
 
-export const ERC20 = { Mana };
+export const ERC20 = { Mana, DetailedERC20 };
```

Comment:
I've used that way because of our `FullNFT` contract name. They are using `ERC721Full` for their NFT full contract. Should I rename both or only ERC20?

Reply:
Hmm, good point. My vote is to rename both.

Reply:
Done.

Reply:
Great, thanks. Resolving this.

Reply:
Sorry but that isn't done. Using the same name of another contract that exists on import hierarchy is broking the project. I've thought that was done because I haven't clean my `build/` folder. That is the reason for the latest CI failures.
I've tried renaming imported Openzeppelin contracts using:
`import {ERC721Full as OpenzeppelinERC721Full} from "openzeppelin-solidity/contracts/token/ERC721/ERC721Full.sol";`
The contract compiles but `truffle` has a colision issue on how `artifacts.require("./ERC721Full.sol");` works.
Refs: https://github.com/trufflesuite/truffle/issues/1087

For quick fix we'll need to use other names than Openzeppelin uses. I've thought to use `FullERC20` and `FullERC721` instead of `ERC20Full` and `ERC721Full`.
What do you think?

Reply:
@marcelomorgado Interesting. We're just using fake versions of instance of ERC20Full and ERC721Full, right? Maybe `MyERC20Full` and `MyERC721Full`?

I like the modifier comes after so that they still sort alphabetically by the main category (not sure that's the most important consideration haha, but I think that's why I like it)

Reply:
We are using them for test and using their ABIs as our supported ERC20/ERC721 interfaces.
I'm like the way they still sort alphabetically too. :+1: :smile: 
Ok, I'll rename to `MyERC20Full` and `MyERC721Full` :+1: 
Thanks for the quick response!

Reply:
Sounds good!

Reply:
Done.

Reply:
Great, thanks! Resolving this